package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sanctuary-benchmark",
+  "description": "Basis for easy benchmarking and sharing of the results",
+  "version": "0.0.0",
+  "main": "index.js",
```

Comment:
The `main` field will be implicitly included in the `files` field. I never tested whether that's also the case when the `main` field is left to its default value. I don't think it will present a problem though, so I'll remove it.